{{cookiecutter.project_slug}}/compose/local/django/celery/beat/start

```diff
@@ -5,4 +5,4 @@ set -o nounset
 
 
 rm -f './celerybeat.pid'
-celery -A config.celery_app beat -l INFO
+celery -A config.celery_app beat -l INFO --scheduler django_celery_beat.schedulers:DatabaseScheduler
```

Comment:
That scheduler comes from the `django_celery_beat` extension published as [an external package](https://pypi.org/project/django-celery-beat/) which isn't installed in cookiecutter-django. There are a couple more things to do [according to the documentation](http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#using-custom-scheduler-classes).

Personally, I would prefer to use the [Django setting](http://docs.celeryproject.org/en/latest/userguide/configuration.html?highlight=CELERYBEAT_SCHEDULER) rather than the command line option, this would avoid non-Docker deployments to forget this (like you forgot the Heroku deployment).

Reply:
Thanks for brining up the Django setting. Didn't know it existed.

Should we add a cookiecutter config option to install `django_celery_beat`? Or assume that someone who wants celery also wants beat?

Reply:
> Should we add a cookiecutter config option to install `django_celery_beat`?

I don't think it's required, sounds a bit overkill.